### PR TITLE
test: Cypress | Flaky fix | Replace static waits - GitSyncedApps_spec.js

### PR DESCRIPTION
**Pull request by Aishwarya-U-R**
## Description
- This PR removes hard wait & flaky fix from spec: ClientSide/Git/GitSync/GitSyncedApps_spec.js

#### Type of change
- Script update (non-breaking change which fixes an issue)

#### How Has This Been Tested?
- [X] Cypress CI runs

#### QA activity:
- [X] Added `Test Plan Approved` label after Cypress tests were reviewed

## Summary by CodeRabbit

- **Tests**
	- Adjusted test logic for currency input handling in regression tests.
	- Renamed test specifications for clarity.
	- Enhanced table widget API binding and pagination tests.
	- Skipped certain test suites for older app imports and admin settings.

- **Chores**
	- Updated the git fetch command in the build workflow for better repository syncing.

- **Documentation**
	- No user-facing documentation changes.

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Git/GitSync/GitSyncedApps_spec.js [hunk 1]
```diff
@@ -118,33 +119,22 @@ describe("Git sync apps", function () {
       "response.body.responseMeta.status",
       201,
     );
+    table.ReadTableRowColumnData(0, 1).then((cellData) => {
+      expect(cellData).to.be.equal("New Config");
+    });
```
[the remark this answers is not in the document]
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/Git/GitSync/GitSyncedApps_spec.js:124]: No, one is in 1st page & other in Cloned page. Its needed.

**Review by sarojsarab — APPROVED** [review]
LGTM

**Pull request merged**